local_pathfinding/local_path.py

```diff
@@ -25,6 +45,13 @@ def __init__(
 
 
 class LocalPath:
+    """The LocalPath sets and updates the local waypoints in between the
```

Comment:
I think you forgot to finish the sentence. Also you can remove "The LocalPath" from the start of the sentence to be more concise